packages/launchpad/src/setup/SelectFwOrBundler.vue

```diff
@@ -30,34 +60,53 @@
   </Select>
 </template>
 
+<script lang="ts">
+
+interface RootOption {
+  name: string;
+  description?: string;
+  id: string;
```

Comment:
Are we not linting Vue files or something? I would have thought `;` would flag as a linting error

Reply:
weird eh! removed anyway